tests/integration-test/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "voxel51-platform-sdk-integration-test",
+  "version": "1.0.0",
```

Comment:
use version `0.1.0`. That's what the `platform-sdk` is, as well as the client libs, etc.